ModulBot/Controllers/MessageController.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Threading.Tasks;
+using Telegram.Bot.Types;
+using Microsoft.AspNetCore.Mvc;
+using Telegram.Bot.Types.Enums;
+using ModulBot.Models;
+using System.Linq;
+using ModulBot.Commands;
+using ModulBot.Cards;
+namespace ModulBot.Controllers
+{
+    [Route("api/message/update")]
+    public class MessageController : Controller
+    {
+        [HttpPost]
+        public async Task<OkResult> Post([FromBody]Update update)
+        {
+            var botClient = await Bot.GetBotClientAsync();
+
+            var user = DataModel.RememberUser(update.Message);
+
+            if (update == null) return Ok();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to 'false'; some subsequent code is never executed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS2583)